orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -71,13 +73,14 @@ def is_disabled(self):
 
 
 class GraphExecutionManager(GraphExecutionInterface):
-    def __init__(self, module, debug_options: DebugOptions, fallback_manager: _FallbackManager):
+    def __init__(self, module, debug_options: DebugOptions, fallback_manager: _FallbackManager, provider_configs = ProviderConfigs):
```

Comment:
What are the current blockers in using ortdevice? Per my knowledge all the build failures were resolved right?

Reply:
@askhade, there are multiple blockers - graph mode not supported for ORT device, data copy and transfer issues, and unsupported ops with 'ORT' backend.